app/views/media_objects/_item_view.html.erb

```diff
@@ -25,9 +25,19 @@ Unless required by applicable law or agreed to in writing, software distributed
 
 <% in_progress = show_progress?(@masterFiles) %>
 <div class="row">
-  <div class="col-sm-8">
+  <div class="col-sm-12">
     <%= render 'workflow_progress' %>
   </div>
+  <div class="col-sm-12" style="margin-bottom: 0.5rem">
```

Comment:
We could probably use `mb-2` Bootstrap class instead of the `margin-bottom` CSS rule here?